content/2015.12.loadtesting.with.oauth.rst

```diff
@@ -0,0 +1,214 @@
+How to load-test a OAuth-protected HTTP API?
+############################################
+
+:lang: en
+:date: 2015-12-02
+:url: loadtesting-a-http-api-which-uses-oauth-for-authentication
+:summary: This article explains how to test the performance of an HTTP API when its access depends on an external OAuth service.
+
+
+Why would you need to run a loadtest?
+=====================================
+
+After deploying an HTTP API you might be wondering how many users or
+requests your deployment can take and what happens after it reaches
+that number.
+
+Knowing the maximum load your service can handle is a great thing to
+know!  It tells you when to start scaling up when your production
+server is close to reach that number of requests per seconds.
+
+
+So, what's so complicated about Oauth?
+======================================
+
+Loadtesting with `OAuth <https://en.wikipedia.org/wiki/OAuth>`_ means
+having an OAuth account, often creating and validating it and then
```

Comment:
having xxx, often xx and xx and then xxx . That sentence is hard to follow. I would split it or use bullet points